src/app/page.tsx

```
@@ -23,30 +34,113 @@ export default function Home() {
+                    <div className={styles.cardButtons}>
+                    <a href="/game/tic-tac-toe" className={styles.playNowButton}>Play Now</a>
+                    <a href="/how-it-works" className={styles.howItWorksButton}>How it works</a>
                     </div>
-                    <div className={styles.githubUrlConatiner}>
-                        <Link href="https://github.com/ajaynegi45/Time-Pass">See Github</Link>
+                </div>
+                </div>
+                {/* Sudoku Game Card*/}
+                 <div className={`${styles.trendingCard} ${styles.cardRight}`}
+                 onClick={()=>handleCardClick("/game/sudoku")}
+                role="link"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<a>
<link>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)